doc/reuse/substitutions.txt

```diff
@@ -1,2 +1,3 @@
 .. |ubuntu-latest-version| replace:: 23.04
 .. |ubuntu-latest-codename| replace:: Lunar Lobster
+.. role:: command(literal)
```

Comment:
Is anything blocking updating the ubuntu-latest values?

Reply:
Just my memory :)